app/templates/components/popular-tags.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="col-md-3">
```

Comment:
I think the component should be decoupled from the grid system here, from a "separation of concerns" standpoint. In a case like this, I try to ask myself "what would happen if I needed to use this component somewhere else?" In this instance, you may not want it to be 3 columns wide; that's reason enough that this component should not know about the grid system, and instead should be inserted _into_ a column rather than including it.